src/jit/emit.h

```diff
@@ -885,14 +885,16 @@ class emitter
         void checkSizes();
 
         union idAddrUnion {
-            // TODO-Cleanup: We should really add a DEBUG-only tag to this union so we can add asserts
-            // about reading what we think is here, to avoid unexpected corruption issues.
+// TODO-Cleanup: We should really add a DEBUG-only tag to this union so we can add asserts
+// about reading what we think is here, to avoid unexpected corruption issues.
 
+#ifndef _TARGET_ARM64_
             emitLclVarAddr iiaLclVar;
```

Comment:
You can check this is how it is.
Originally I though that _idReg3 and _idReg4 weren't defined for 32-bit, but they are.